extensions/wikia/GlobalNavigation/scripts/GlobalNavigationNotifications.js

```diff
@@ -33,89 +33,66 @@ require(
 				this.unreadCount = parseInt(this.$notificationsCount.html(), 10);
 
 				this.$notifications
-					.mouseenter( this.proxy( this.updateCounts ) )
-					.mouseenter( this.proxy( this.fetchForCurrentWiki ) );
+					.mouseenter(this.proxy(this.updateCounts))
+					.mouseenter(this.proxy(this.fetchForCurrentWiki));
 
-				this.$wallNotifications.add( $('#pt-wall-notifications') )
-					.on('click', '.notifications-markasread', this.proxy( this.markAllAsReadAllWikis ));
+				this.$wallNotifications.add($('#pt-wall-notifications'))
+					.on('click', '.notifications-markasread', this.proxy(this.markAllAsReadAllWikis));
```

Comment:
consider: `this.markAllAsReadAllWikis.bind(this);`

I am not changing it as I am not aware of all side effects